source/Outfit.cpp

```diff
@@ -279,6 +279,8 @@ void Outfit::Load(const DataNode &node)
 			// of blocking the sale of the outfit until the ammo is sold first.
 			ammo = make_pair(GameData::Outfits().Get(child.Token(1)), 0);
 		}
+		else if(child.Token(0) == "ammo resupply" && child.Size() >= 3)
+			resuppliedAmmo.emplace_back(GameData::Outfits().Get(child.Token(1)), child.Value(2));
```

Comment:
Why not use a syntax like licenses where the values are children?
```
"ammo resupply"
    <ammo outfit> <value>
    <other ammo outfit> <value>
```
?
And, can we just not support have multiple values for the same resupplyable ammo outfit on a single outfit?
It has no meaning since all but the lowest value are being ignored and it is the source of unnecessary complexity in other parts of this code.

Reply:
> > can we just not support have multiple values for the same resupplyable ammo outfit on a single outfit?
> 
> We'll still need to do so across all outfits.

That's fine.